test_container fails

<details>
<summary>Test logs</summary>

```
=================================== FAILURES ===================================
___________________________ test_container_with_tag ____________________________

get_version = <function get_version.<locals>.__call__ at 0x7f1b7ee98720>

    async def test_container_with_tag(get_version):
>     update_time = await get_version("bitnami/mongodb:5.0", {
        "source": "container",
        "container": "bitnami/mongodb:5.0",
      })

tests/test_container.py:19:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
tests/conftest.py:57: in __call__
    newvers = await run(entries)
              ^^^^^^^^^^^^^^^^^^
tests/conftest.py:50: in run
    results, _has_failures = await main.run(result_coro, runner_coro)
                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
nvchecker/__main__.py:109: in run
    return await result_fu
           ^^^^^^^^^^^^^^^
nvchecker/core.py:443: in process_result
    logger.exception('error processing result', result=r)
/usr/lib/python3.13/site-packages/structlog/_native.py:47: in exception
    return self.error(event, *args, **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/usr/lib/python3.13/site-packages/structlog/_native.py:164: in meth
    return self._proxy_to_logger(
/usr/lib/python3.13/site-packages/structlog/_base.py:222: in _proxy_to_logger
    args, kw = self._process_event(method_name, event, event_kw)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/usr/lib/python3.13/site-packages/structlog/_base.py:173: in _process_event
    event_dict = proc(self._logger, method_name, event_dict)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
nvchecker/core.py:441: in process_result
    r1 = _process_result(r)
         ^^^^^^^^^^^^^^^^^^
nvchecker/core.py:369: in _process_result
    logger.error('unexpected error happened',
/usr/lib/python3.13/site-packages/structlog/_native.py:164: in meth
    return self._proxy_to_logger(
/usr/lib/python3.13/site-packages/structlog/_base.py:222: in _proxy_to_logger
    args, kw = self._process_event(method_name, event, event_kw)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/usr/lib/python3.13/site-packages/structlog/_base.py:173: in _process_event
    event_dict = proc(self._logger, method_name, event_dict)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
tests/conftest.py:87: in proc
    raise exc
nvchecker/util.py:314: in run_one
    version = await self.func(
nvchecker_source/container.py:162: in get_version
    return await cache.get(key, get_container_tag_update_time)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
nvchecker/util.py:253: in get
    r = await fu
        ^^^^^^^^
nvchecker_source/container.py:123: in get_container_tag_update_time
    res = await session.get(url, headers=headers)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
nvchecker/httpclient/base.py:49: in get
    return await self.request(
nvchecker/httpclient/base.py:79: in request
    return await self.request_impl(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <nvchecker.httpclient.tornado_httpclient.TornadoSession object at 0x7f1b7eea7380>
url = 'https://registry-1.docker.io/v2/bitnami/mongodb/manifests/5.0'

    async def request_impl(
      self, url: str, *,
      method: str,
      proxy: Optional[str] = None,
      headers: Dict[str, str] = {},
      follow_redirects: bool = True,
      params = (),
      json = None,
      body = None,
      verify_cert: bool = True,
    ) -> Response:
      kwargs: Dict[str, Any] = {
        'method': method,
        'headers': headers,
        'request_timeout': self.timeout,
        'follow_redirects': follow_redirects,
        'validate_cert': verify_cert,
      }

      if body:
        # By default the content type is already 'application/x-www-form-urlencoded'
        kwargs['body'] = body
      elif json:
        kwargs['body'] = _json.dumps(json)
      kwargs['prepare_curl_callback'] = setup_curl

      if proxy:
        host, port = proxy.rsplit(':', 1)
        kwargs['proxy_host'] = host
        kwargs['proxy_port'] = int(port)

      if params:
        q = urlencode(params)
        url += '?' + q

      r = HTTPRequest(url, **kwargs)
      res = await AsyncHTTPClient().fetch(
        r, raise_error=False)
      err_cls: Optional[type] = None
      if res.code >= 500:
        err_cls = TemporaryError
      elif res.code >= 400:
        err_cls = HTTPError
      if err_cls is not None:
>       raise err_cls(
          res.code, res.reason, res
        )
E       nvchecker.httpclient.base.HTTPError: (404, 'Not Found', HTTPResponse(_body=None,_error_is_response_code=True,buffer=<_io.BytesIO object at 0x7f1b6fd78220>,code=404,effective_url='https://registry-1.docker.io/v2/bitnami/mongodb/manifests/5.0',error=HTTP 404: Not Found,headers=<tornado.httputil.HTTPHeaders object at 0x7f1b6fe70cb0>,reason='Not Found',request=<tornado.httpclient.HTTPRequest object at 0x7f1b7eee2850>,request_time=0.09912824630737305,start_time=1762562655.2612903,time_info={'queue': 5.245208740234375e-06, 'namelookup': 0.0, 'connect': 0.0, 'appconnect': 0.0, 'pretransfer': 8.1e-05, 'starttransfer': 0.09882, 'total': 0.09894, 'redirect': 0.0}))

nvchecker/httpclient/tornado_httpclient.py:97: HTTPError
=========================== short test summary info ============================
FAILED tests/test_container.py::test_container_with_tag - nvchecker.httpclien...
================== 1 failed, 132 passed, 15 skipped in 51.93s ==================
```

</details>

The 5.0 tag in bitnami/mongodb appears gone (https://hub.docker.com/r/bitnami/mongodb/tags). Per https://github.com/lilydjwg/nvchecker/pull/246, we may need to find another non-multi-arch image.

--- a/tests/test_container.py
+++ b/tests/test_container.py
@@ -16,12 +16,12 @@ async def test_container(get_version):
   }) == "linux"
 
 async def test_container_with_tag(get_version):
-  update_time = await get_version("bitnami/mongodb:5.0", {
+  update_time = await get_version("amd64/busybox:1.31.1", {
     "source": "container",
-    "container": "bitnami/mongodb:5.0",
+    "container": "amd64/busybox:1.31.1",
   })
   # the update time is changing occasionally, so we can not compare the exact time, otherwise the test will be failed in the future
-  assert datetime.date.fromisoformat(update_time.split('T')[0]) > datetime.date(2023, 12, 1)
+  assert datetime.date.fromisoformat(update_time.split('T')[0]) > datetime.date(2020, 1, 1)
 
 async def test_container_with_tag_and_multi_arch(get_version):
   update_time = await get_version("hello-world:linux", {

